web/models/run.ex

```diff
@@ -5,13 +5,15 @@ defmodule Gatherto.Run do
     field :title, :string
     field :description, :string
     field :time, Ecto.DateTime
+    field :minimum_distance, :integer
+    field :maximum_distance, :integer
     belongs_to :created_by, Gatherto.Athlete, foreign_key: :athlete_id
     many_to_many :athletes, Gatherto.Athlete, join_through: Gatherto.AthleteRun
 
     timestamps()
   end
 
-  @required_params [:title, :description, :time]
+  @required_params [:title, :description, :time, :minimum_distance, :maximum_distance]
 
```

Comment:
I like that we're validating with ```validate_distance_range``` that the  but I'm not sure if we should make these required. I can see runners wanting to set a pace range and a location but being flaky about the distance. Especially since some workouts focus more on time and pace (and not on distance). I could also see runners wanting to add in a minimum but leaving the max wide open. I'm wondering if we should just validate the distance range when they've been entered, but default to nil if not and make it acceptable to have either a min and nil, a max and nil, a min that's < max , or nil and nil - what do you think?